capstone-ui/src/Components/ResultsBox/ResultsBox.jsx

```diff
@@ -0,0 +1,75 @@
+import * as React from "react"
+import { useParams } from "react-router-dom"
+import electionResults from "../../Data/2020presidential.json"
+
+export default function ResultsBox( { locationLevel, countyFIPS } ) {
```

Comment:
Which part? locationLevel is used by the pages which call the ResultsBox component to determine whether it needs results for the country, state, or a county. countyFIPS is used because it needs the specific county code to get the results for the county in the state. This was fully implemented in the next PR.